src/sound_openal.cpp

```diff
@@ -457,18 +536,10 @@ class OpenALSoundManager: public ISoundManager
 	bool loadSoundData(const std::string &name,
 			const std::string &filedata)
 	{
-		// The vorbis API sucks; just write it to a file and use vorbisfile
-		// TODO: Actually load it directly from memory
-		std::string basepath = porting::path_user + DIR_DELIM + "cache" +
-				DIR_DELIM + "tmp";
-		std::string path = basepath + DIR_DELIM + "tmp.ogg";
```

Comment:
I think that would better fit into some client specific path initialisation function, and not into the sound_openal.cpp file. If we edited this function to remove basepath, it would be executed for every sound file, that's not neccessary. Once when minetest is started is enough.

Do you want to make a patch for it?